<ui version="4.0" >
 <class>KpkSettings</class>
 <widget class="QWidget" name="KpkSettings" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>615</width>
    <height>486</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Settings</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3" >
   <property name="margin" >
    <number>0</number>
   </property>
   <property name="spacing" >
    <number>0</number>
   </property>
   <item row="0" column="0" >
    <widget class="QGroupBox" name="groupBox" >
     <property name="title" >
      <string>KPackageKit Settings</string>
     </property>
     <layout class="QGridLayout" name="gridLayout" >
      <item row="0" column="0" >
       <widget class="QLabel" name="intervalL" >
        <property name="text" >
         <string>Check for Updates:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" colspan="2" >
       <widget class="QComboBox" name="intervalCB" >
        <property name="currentIndex" >
         <number>-1</number>
        </property>
       </widget>
      </item>
      <item row="1" column="0" >
       <widget class="QLabel" name="label_3" >
        <property name="enabled" >
         <bool>true</bool>
        </property>
        <property name="text" >
         <string>Automatically Install:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" colspan="2" >
       <widget class="QComboBox" name="autoCB" >
        <property name="enabled" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0" >
    <widget class="QGroupBox" name="groupBox_2" >
     <property name="title" >
      <string>Smart Tray Icon</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2" >
      <item row="0" column="0" >
       <widget class="QCheckBox" name="notifyUpdatesCB" >
        <property name="enabled" >
         <bool>true</bool>
        </property>
        <property name="text" >
         <string>Notify when updates are avaliable</string>
        </property>
        <property name="checked" >
         <bool>true</bool>
        </property>
        <property name="tristate" >
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0" >
       <widget class="QCheckBox" name="notifyLongTasksCB" >
        <property name="enabled" >
         <bool>false</bool>
        </property>
        <property name="text" >
         <string>Notify when long tasks have been completed</string>
        </property>
        <property name="checked" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0" >
    <spacer name="verticalSpacer" >
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType" >
      <enum>QSizePolicy::Ignored</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="0" >
    <widget class="QGroupBox" name="originGB" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Expanding" hsizetype="Expanding" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title" >
      <string>Origin of Packages</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_4" >
      <item row="0" column="0" >
       <widget class="QListView" name="originLW" />
      </item>
      <item row="1" column="0" >
       <widget class="QCheckBox" name="showOriginsCB" >
        <property name="text" >
         <string>&amp;Show origins of debug and development packages</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
